animation_nodes/nodes/geometry/test_intersection.py

```diff
@@ -0,0 +1,107 @@
+from mathutils import Vector
+from unittest import TestCase
+from . c_utils import (
+                       intersect_LineLine_Single,
```

Comment:
please indent these lines only once (I know I don't do this myself everytime, I'm not sure yet what the best way is, to import many things)